packages/legacy/core/App/components/misc/InfoBox.tsx

```diff
@@ -20,30 +21,40 @@ export enum InfoBoxType {
   Error,
 }
 
-interface BifoldErrorProps {
```

Comment:
This must have been a copy paste mistake from way back. It's not exported, so I thought it safe to rename correctly.